pkg/appfs/server.go

```diff
@@ -99,19 +101,52 @@ func (g gzipReadCloser) Close() error {
 	return nil
 }
 
+type cacheEntry struct {
+	content []byte
+	headers swift.Headers
+}
+
+var cache *lru.Cache[string, cacheEntry]
+var initCacheOnce sync.Once
+
 // NewSwiftFileServer returns provides the apps.FileServer implementation
 // using the swift backend as file server.
 func NewSwiftFileServer(conn *swift.Connection, appsType consts.AppType) FileServer {
+	initCacheOnce.Do(func() {
+		c, err := lru.New[string, cacheEntry](128)
```

Comment:
Why such a small cache? Don't we have more konnectors and applications already?

Reply:
Well, it can take a lot of RAM. I prefer to start small and see later to increase.